index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey microverse website</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <h1 class="heading-primary">Microverse Survey Process</h1>
+      </header>
+      <div id="container">
+        <section class="form-wrap">
+          <h2>Survey</h2>
+          <p>It's free and only takes a minute</p>
+          <form>
+            <div class="form-group">
+              <label for="name">Name</label>
+              <input type="text" id="name" placeholder="Enter your name" />
+            </div>
+            <div class="form-group">
+              <label for="sex">Sex</label>
+              <select name="sex" id="sex">
+                <option value="male" selected>Male</option>
+                <option value="female">Female</option>
+                <option value="other">Other</option>
+              </select>
```

Comment:
- [ ] According to the project's requirements, you have to:
- Use the GitHub inline comments feature at least once to leave review comments. For this project, kindly leave the comment under a line or block of code that fits the context the comment is about. This not only makes the comment complete but understandable as well. I will try to invite you to check this [lesson on how to add inline comments](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request).
- You have to write at least 2 comments in the review.

- [ ] You've made a polite request [here](https://github.com/bernyhas/Form-Survey/pull/1#issuecomment-875474864), but your suggestion lacks a `why`. Include an explanation of why you think the student should improve the semantic HTML tags. 

Please always include an explanation, a "why", for each requested change. For example:
- This is a well-explained request: 
  Kindly remove the `header` tag from being a child of the `main` tag, this helps to improve the semantic correctness of the HTML structure and making the page more accessible. 
- This request lacks a "why": 
Make sure you improve your Html semantic tag.

- [ ] A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary like this current review. :wink:
This is an example inline comment. :slightly_smiling_face:
You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:
To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.